### PR TITLE
Update rule titles

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update rule titles

#### Rationale:

- Based on #4243

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_maxclassrepeat/rule.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ documentation_complete: true
 
 prodtype: rhel7,rhel8,fedora,ol7,ol8,rhv4
 
-title: 'Set Password to Maximum of Consecutive Repeating Characters from Same Character Class'
+title: 'Ensure PAM Enforces Password Requirements - Maximum of Consecutive Repeating Characters from Same Character Class'
```
Comment (shawndwells) [inline linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_maxclassrepeat/rule.yml]:
```suggestion
title: 'Ensure PAM Enforces Password Requirements - Maximum Consecutive Repeating Characters from Same Character Class'
```

**Comment by shawndwells** [comment]
@openscap-ci retest this please

File: linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_maxclassrepeat/rule.yml (hunk 1 above)
Comment (yuumasato) [reply linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_maxclassrepeat/rule.yml]: That is a interesting suggest and commit feature by GitHub.

**Comment by shawndwells** [comment]
On 4/9/19 3:42 AM, Watson Yuuma Sato wrote:
>
> That is a interesting suggest and commit feature by GitHub.
>

Comes in handy for small pitpicks!

**Pull request merged**